cif_pow.dic

```diff
@@ -121,6 +121,1229 @@ save_pd_amorphous.phase_id
+    used, rather than a complete enumeration of the value of the background
+    calculated at every data point; although doing so is still possible -  see
+    _pd_proc.intensity_bkg_calc.
+
+    The computed background values should include all normalization corrections,
+    and thus are specified on the same scale as the observed intensities
+    (_pd_meas.counts_* or _pd_meas.intensity_*, and _pd_calc.intensity_total).
+
+    If more than one type of background is specified for a particular
+    diffractogram, then the it is assumed they are linearly additive.
+;
+    _name.category_id             PD_GROUP
+    _name.object_id               PD_BACKGROUND
+
+    loop_
+      _category_key.name
+         '_pd_background.diffractogram_id'
+         '_pd_background.id'
+
+save_
+
+save_pd_background.air_or_thermal_diffuse_coeff_1
+
+    _definition.id
+        '_pd_background.air_or_thermal_diffuse_coeff_1'
+    _definition.update            2023-01-14
+    _description.text
+;
+    The value of the first coefficient used in an equation representing the
+    background due to thermal diffuse scatering and/or air scattering, in a
+    calculated diffractogram. This equation can account for background
+    contributions at both high and low q. The first coefficient accounts for
+    background contributions that increase with q. Must be given with a
+    _pd_background.air_or_thermal_diffuse_order value.
+
+    The background equation is of the form:
+
+        bkg = Sum( C1~j~ * (q~2*j~/j!) + C2~j~ * (j!/q~2*j~), j=0:N)
+
+    where C1 and C2 represent _pd_background.air_or_thermal_diffuse_coeff_1 and
+    coeff_2, respectively, j is the order of the equation as given in
+    _pd_background.air_or_thermal_diffuse_order, and q is the magnitude of the
+    diffraction vector, defined as
+
+        q = 4 * Pi * sin(theta) / lambda
+
+    where theta is the diffraction angle and lambda is the wavelength of the
+    incident radiation in angstroms.
+;
+    _name.category_id             pd_background
+    _name.object_id               air_or_thermal_diffuse_coeff_1
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _units.code                   none
+
+save_
+
+save_pd_background.air_or_thermal_diffuse_coeff_1_su
+
+    _definition.id
+        '_pd_background.air_or_thermal_diffuse_coeff_1_su'
+    _definition.update            2023-01-14
+    _description.text
+;
+    Standard uncertainty of pd_background.air_or_thermal_diffuse_coeff_1.
+;
+    _name.category_id             pd_background
+    _name.object_id               air_or_thermal_diffuse_coeff_1_su
+    _name.linked_item_id
+        '_pd_background.air_or_thermal_diffuse_coeff_1'
+    _units.code                   none
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_pd_background.air_or_thermal_diffuse_coeff_1_list
```

Comment:
As a rule you don't state the type of the data name in the data name itself as that would be clear from the value anyway. So perhaps `_pd_background.air_or_thermal_diffuse_coeffs` would be better.